tests/platform_tests/api/test_fan_drawer.py

```diff
@@ -40,6 +41,10 @@ def setup(self, platform_api_conn):
             except:
```

Comment:
LGTM is complaining about this. Change to `except Exception:`

Reply:
changed as suggested, fixed